src/staticfile/integration/deploy_enable_http2_test.go

```diff
@@ -0,0 +1,124 @@
+package integration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/cloudfoundry/libbuildpack/cutlass"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("deploy a HTTP/2 app", func() {
+	var app *cutlass.App
+	var app_name string
+	AfterEach(func() {
+		if app != nil {
+			app.Destroy()
+		}
+		app = nil
+		app_name = ""
+	})
+
+	BeforeEach(func() {
+		apiVersionString, err := cutlass.ApiVersion()
+		Expect(err).To(BeNil())
+		apiVersion, err := semver.Make(apiVersionString)
+		Expect(err).To(BeNil())
+		apiHasHttp2, err := semver.ParseRange("> 2.172.0")
```

Comment:
Do all v3 endpoints support HTTP/2 ? This semver will pass all v3 endpoints.

However, answering/fixing this is probably not required for this PR as we don't really check this in other parts of the test

Reply:
We need [capi 1.118.0](https://github.com/cloudfoundry/capi-release/releases/tag/1.118.0) which has api version 2.172.0 and 3.107.0.  We only need to check 2.x.x version as capi releases always bump api version.  We could switch it to 3.107.0 if that makes it less confusing

Reply:
No, that makes sense. Thanks for explaining.

I didn't make the connection that you don't care about v2 vs v3 but instead you care about the underlying CAPI functionality. I also didn't make the connection that ensuring the api version 2.172.0 also guarantees the specific v3 endpoints you require.

Reply:
Oh, but shouldn't the `>` read `>=`?

Reply:
Yes you're right.  I shall update that